### ISSUE TITLE
Missing some basic_selections for new CSM 9e codex

### ISSUE THREAD

**Issue by JLoach** [maybe-hello-world/40k-roster-formatter#53]
There are a few required wargear selections missing due to renames, etc in the new codex:

```
   Possessed:
     - Horrifying Mutations
+    - Hideous mutations

+  Master of Possession:
+    - Staff of Possession
+
+  Venomcrawler:
+    - Soulflayer tendrils and claws
```

I would be happy to help out keeping that up to date and adding others I spot if you give me access to file a PR.

**Comment by maybe-hello-world** [comment]
Hello!
Sorry for the long answer and thanks for the suggestion. The usual way to create a PR is to fork the repo, make changes and file a PR. Unfortunately, other ways (AFAIK) require write access to the repo.

I'll be happy to accept PR if you will create one, or make changes by myself if it's too much work for several lines and you don't want to :D

**Comment by JLoach** [comment]
Ok, sure. I am more used to a branch oriented approach, but I can fork and file a PR off that.

**Comment by JLoach** [comment]
Filed a PR for these first few additions.

**Comment by maybe-hello-world** [comment]
Closed by #54
